proof/crefine/AARCH64/Schedule_C.thy

```diff
@@ -100,6 +102,26 @@ lemma Arch_switchToThread_ccorres:
   apply (clarsimp simp: typ_heap_simps ctcb_relation_def carch_tcb_relation_def)
   done
 
+lemma all_invs_but_ct_idle_or_in_cur_domain'_pspace_aligned':
+  "all_invs_but_ct_idle_or_in_cur_domain' s \<Longrightarrow> pspace_aligned' s"
```

Comment:
I think there's an abbreviation for this, `invs_no_cicd'`? A bunch of lemmas also use the abbreviated form in their name, as it's somewhat long-winded.

Reply:
I generally hate these abbreviations. It's just another bunch of stuff to remember. For example, `invs_psp_aligned` versus `invs_distinct` or whatever. It's rather horrible. But I can do this here.

Reply:
The irony is I can live with the longer versions, but am also trying to keep consistent with what the rest of the proofs are doing. @lsf37 do you have any thoughts on this? I don't really have a long history with `invs_no_cicd'`.

Reply:
I think we should standardise on the short name everywhere (`invs_no_cicd'`) and have the explanation in a comment in the description. Having both with abbreviation/definition is indeed annoying and a source of confusion. (Also agree with the annoyance for `invs_psp_aligned` vs `invs_distinct` etc -- those are very old names and it would be nice to go to a standardised naming scheme for them -- for those I wouldn't mind the full names, they are not quite as long as `all_invs_but_ct_idle_or_in_cur_domain`).

That should be a separate PR/issue, though. This one might actually be relatively easy/fast to do with a script, since it's pure renaming.